studio/src/aem/aem-fragments.js

```diff
@@ -227,6 +227,12 @@ class AemFragments extends LitElement {
 
     async publishFragment() {
         await this.#aem.sites.cf.fragments.publish(this.fragment);
+        setTimeout(async () => {
+            const latest = await this.#aem.sites.cf.fragments.getById(
+                this.fragment.id,
+            );
+            aemFragmentCache.get(this.fragment.id)?.refreshFrom(latest);
+        }, 4000);
```

Comment:
are we sure this will work? worst case we have stale cache?

Reply:
I'm not a fun of arbitrary `setTimeout`. 
If we want to refresh status, let's support the official APIs: https://developer.adobe.com/experience-cloud/experience-manager-apis/api/stable/sites/

![image](https://github.com/user-attachments/assets/8c703ff8-d1e8-4319-88cc-4135427b3859)